tests/controls/windowtest.cpp

```diff
@@ -143,11 +192,29 @@ TEST_CASE_METHOD(WindowTestCase, "Window::FocusEvent", "[window]")
 
 TEST_CASE_METHOD(WindowTestCase, "Window::Mouse", "[window]")
 {
+    wxInitAllImageHandlers();
     wxCursor cursor(wxCURSOR_HAND);
     m_window->SetCursor(cursor);
 
     CHECK(m_window->GetCursor().IsOk());
 
+#if defined( __WXMSW__ ) || defined( __WXOSX__ )
+    wxCursor resCursor1( "horse" );
+    CHECK( resCursor1.IsOk() );
+#endif
+#ifdef __WXGTK__
+    // According to documentation cursors on X/GTK are different than in Windows/OSX.
+    // On X/GTK, using the constructor with wxString parameter is loading the cursor
+    // from the external file and not creating it from embedded bits/resources
+    // Therefore different path are used for creating embedded cursors
+    wxCursor resCursor1( (const char *) horse, 32, 32, 16, 0, nullptr, wxWHITE, wxBLACK );
+    CHECK( resCursor1.IsOk() );
+#endif
+#ifdef __WXOSX__
+    // Check the cur file
+    wxCursor resCursor2( "bullseye" );
+    CHECK( resCursor2.IsOk() );
+#endif
```

Comment:
I think this test should go into `tests/image/image.cpp` where we already have some similar tests for `wxCursor` and have a test XPM too, it doesn't seem right to have it here.

Reply:
I will move it. But the `wxCursor` constructor accepts XBM (X BitMap), not XPM (X PixMap) so it can't reuse `g_testfiles[1]`.

Reply:
Looks like the horse xbm data is the contents of `horse.cur`, so I'll just read that file to a buffer.